storyandpassage/snowman/snowman_storyandpassage_twee.txt

```diff
@@ -0,0 +1,17 @@
+:: StoryTitle
+Story and Passage API in Snowman
+
+:: Start
+The title of this story is "<%= window.story.name %>."
```

Comment:
@videlais 
Is there a reason why you used `window.story.name` instead of just `story.name`?
The same goes for `window.story.passage()` instead of just `story.passage()`

note: I understand the need to include the `window` reference when defining `window.setup` because it helps to scope the new 'variable' (window property) but I don't see the point of using `window` when referencing the properties of that 'global' scope object.